make.sh

```diff
@@ -38,7 +38,7 @@ cd -
 
 echo "Adding bindata"
 
-go-bindata $BINDATA_ARGS config.json db/migrations/ $(find ./public -type d -print)
+go-bindata $BINDATA_ARGS config.json db/migrations/ $(find public/* -type d -print)
```

Comment:
Also added this fix, because paths with trailing slashes are being aborted with 404 for some reason (was struggling with building & running locally):
https://github.com/ansible-semaphore/semaphore/blob/master/api/router.go#L115